system/scripts/zone/ais/basic_monster.cs

```diff
@@ -14,6 +14,7 @@ public class BasicMonsterAiScript : AiScript
 	protected override void Setup()
 	{
 		During("Idle", CheckEnemies);
+		During("Attack", CheckEnemies);
```

Comment:
By adding this event callback, as well as a call for `CheckEnemies` to the `CheckTarget` function, it would be called twice right now. Probably not an issue, but unnecessary.